### ISSUE TITLE
Verbose mode

### ISSUE THREAD

**Issue by sassanh** [python-poetry/install.python-poetry.org#141]
When for whatever reason an error keeps occurring and needs some back and forth, it's a bit inconvenient to run the curl command, copy the log location, and tail it repeatedly. It would be nice if we had an environment variable to make the script print everything it writes in logs directly to stdout.

Also when pip it is retrying a url, it may take few minutes until the process finishes and prints the location of the log file, while if it printed the errors in stdout simultaneously (when the environment variable is set) one could immediately see it is a network error, saving time.

**Comment by ctcampbell** [comment]
Agreed, especially when using this script in a Docker build it's not super simple to grab the log file.
